guides/doc-Provisioning_Guide/topics/Cloud-Amazon_EC2.adoc

```diff
@@ -247,15 +274,27 @@ EOS
 . Click the *Templates* tab, and from the *Finish Template* list, select your finish template.
 . Navigate to *Hosts* > *Create Host* and enter the information about the host that you want to create.
 . Click the *Parameters* tab and navigate to *Host parameters*.
+ifeval::["{Build}" == "satellite"]
 . In *Host parameters*, click the *Add Parameter* button three times to add three new parameter fields. Add the following three parameters:
+endif::[]
+ifeval::["{Build}" == "foreman"]
+. In *Host parameters*, click the *Add Parameter* button two times to add two new parameter fields. If you use the Katello plugin, add a third parameter field. Add the following parameters:
+endif::[]
 .. In the *Name* field, enter `remote_execution_ssh_keys`. In the corresponding *Value* field, enter the output of `cat /usr/share/foreman/.ssh/id_rsa_foreman_proxy.pub`.
 .. In the *Name* field, enter `remote_execution_ssh_user`. In the corresponding *Value* field, enter `ec2-user`.
+ifeval::["{Build}" == "satellite"]
 .. In the *Name* field, enter `activation_keys`. In the corresponding *Value* field, enter your activation key.
+endif::[]
+ifeval::["{Build}" == "foreman"]
+.. If you use the Katello plugin, in the *Name* field, enter `activation_keys`. In the corresponding *Value* field, enter your activation key.
+endif::[]
 . Click *Submit* to save the changes.
```

Comment:
It is actually more complicated - remote_execution stuff only appears when Remote Execution plugin is installed. Probably just keep it as-is and add a NOTE that will explain some might be missing when ReX or Katello is not installed.

Reply:
Ha, this is why we are doing this. Thank you so much. I had no idea.

Reply:
This is displayed as unchanged but if you look in the file I have made the changes.